src/OpenDB/src/lefout/lefout.cpp

```diff
@@ -114,26 +201,76 @@ void lefout::writeHeader(dbLib* lib)
   if (right_bus_delimeter == 0)
     right_bus_delimeter = ']';
 
-  char hier_delimeter = lib->getHierarchyDelimeter();
-
   if (hier_delimeter == 0)
     hier_delimeter = '|';
 
-  fprintf(_out, "VERSION %s ;\n", tech->getLefVersionStr());
-  fprintf(_out,
-          "NAMESCASESENSITIVE %s ;\n",
-          tech->getNamesCaseSensitive().getString());
+  writeVersion(tech->getLefVersionStr());
+  writeNameCaseSensitive(tech->getNamesCaseSensitive());
+  writeBusBitChars(left_bus_delimeter, right_bus_delimeter);
+  writeDividerChar(hier_delimeter);
+  writePropertyDefinitions(lib);
+
+  if (lib->getLefUnits()) {
+    writeUnits(lib->getLefUnits());
+  }
+}
```

Comment:
Didn't you remove this to be consistent with the other header?

Reply:
This is for the lib writer which included the units header. I did not want to change this behavior. The lef writer for blocks won't include units unless that turns out to be needed.